packages/base/all/vendor-config-onl/src/python/onl/platform/base.py

```diff
@@ -609,6 +609,10 @@ class OnlPlatformPortConfig_6x400_48x50(object):
     PORT_COUNT=54
     PORT_CONFIG="6x400 + 48x50"
 
+class OnlPlatformPortConfig_6x10_8x1_4x1(object):
+    PORT_COUNT=18
+    PORT_CONFIG="6x10 + 8x1 + 4x1"
+
```

Comment:
I’d put this in a separate commit.

Reply:
If this is put in a separate commit and it is merged after this PR(#790), then there will be build error.